From 1b4c485427afc6d17a04159b64dc966dbab1b4cc Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 11 Jul 2019 06:07:40 +0200
Subject: [PATCH 1/1] efi_loader: duplicate IMAGE_SUBSYSTEM_EFI_APPLICATION

IMAGE_SUBSYSTEM_EFI_APPLICATION is already defined in
include/asm-generic/pe.h which is included in include/pe.h. We should
not define it twice.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/pe.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/pe.h b/include/pe.h
index 36e1908b7e..c7aeb2dcb6 100644
--- a/include/pe.h
+++ b/include/pe.h
@@ -48,7 +48,6 @@ typedef struct _IMAGE_DOS_HEADER {
 
 #define IMAGE_NT_OPTIONAL_HDR32_MAGIC	0x10b
 #define IMAGE_NT_OPTIONAL_HDR64_MAGIC	0x20b
-#define IMAGE_SUBSYSTEM_EFI_APPLICATION	10
 
 typedef struct _IMAGE_FILE_HEADER {
 	uint16_t Machine;
-- 
2.20.1

